compliance_checker/cf/cf.py

```diff
@@ -491,11 +491,15 @@ def check_units(self, ds):
             # skip string type vars
             if v.dtype.char == 'S':
                 continue
-
+            
+            # skip quality control vars
+            if hasattr(v, 'flag_meanings'):
+                continue
+            
             units = getattr(v, 'units', None)
 
             # 1) "units" attribute must be present
-            presence = Result(BaseCheck.HIGH, units is not None, ('units', k, 'present'))
+            presence = Result(BaseCheck.MEDIUM, units is not None, ('units', k, 'present'))
```

Comment:
I don't think this priority reduction is appropriate.  The [CF document states](http://cfconventions.org/Data/cf-conventions/cf-conventions-1.6/build/cf-conventions.html#units):

> The units attribute is required for all variables that represent dimensional quantities...

The use of "required" in this phrase means it is high priority.

I do agree that we need to better identify the variables which this phrase applies to.